src/NzbDrone.Core/Datastore/TableMapping.cs

```diff
@@ -168,7 +172,8 @@ private static void RegisterMappers()
 
         private static void RegisterProviderSettingConverter()
         {
-            var settingTypes = typeof(IProviderConfig).Assembly.ImplementationsOf<IProviderConfig>();
+            var settingTypes = typeof(IProviderConfig).Assembly.ImplementationsOf<IProviderConfig>()
+                .Where(x => !x.ContainsGenericParameters);
```

Comment:
With credits separated, is this needed anymore?

Reply:
I think so, it's this that causes grief I think:
```
 public class TMDbSettingsBase<TSettings> : IProviderConfig
```
It doesn't like the generic